src/serialize.h

```diff
@@ -256,8 +260,9 @@ template<typename Stream> inline void Unserialize(Stream& s, Span<unsigned char>
 template<typename Stream> inline void Serialize(Stream& s, bool a)    { char f=a; ser_writedata8(s, f); }
 template<typename Stream> inline void Unserialize(Stream& s, bool& a) { char f=ser_readdata8(s); a=f; }
 
-
-
+// Non serialized empty value
+template<typename Stream> inline void Serialize(Stream& s, NonSerializedEmptyValue a) { }
+template<typename Stream> inline void Unserialize(Stream& s, NonSerializedEmptyValue& a) { a = NonSerializedEmptyValue{}; }
```

Comment:
prefer empty body, now it's pure non-sense

Reply:
@bvbfan - it doesn't prevent someone from accidentally modifying the value (or worse the static values) . 
This resets always to the expected value post serialisation attempt to avoid surprises.

Not opinionated -- but I'm curious on your how you measure "purity" of non-sense ;)

Reply:
Indeed, that's the intention. But forces someone to think about it when someone adds something that requires init into the class for whatever reason instead of it silently being ignored. 

Either way, compiler will likely consider it empty block until precursors change. Remove if that's make it appear cleaner - not opinionated on this.